### DISCUSSION TITLE
blog/why-meshtastic-uses-managed-flood-routing/

### DISCUSSION THREAD

**Discussion by giscus** [meshtastic/Discussions#14] [Blog Post Comments]
# blog/why-meshtastic-uses-managed-flood-routing/

Why one of the simplest solutions is best for Meshtastic

https://meshtastic.org/blog/why-meshtastic-uses-managed-flood-routing/

**Comment by TyIsI** [comment]
As I'm currently researching full mesh options, do the Meshtastic protocols have for high convergence situations?

Where I live, there currently aren't a lot of nodes, but with high convergence, that would create the risk of easily running into hop limits if there are too many radios in one particular area and/or create asymmetric paths.

**Comment by markerase** [comment]
I really appreciate all the valuable insights shared in this article. It offers plenty of practical knowledge that I find quite useful. I have bookmarked this wonderful post immediately. https://cse.google.co.uk/url?q=https://markerase.com/
